lib/src/components/NxFilterInput/NxFilterInput.scss

```diff
@@ -35,3 +35,17 @@
     margin-right: 8px;
   }
 }
+
+.nx-filter-input--datalist {
+  .nx-text-input__box {
+    padding-bottom: 0;
+  }
+
+  .nx-text-input__input {
+    padding: 0 0 8px 0;
+  }
+
+  .nx-icon--filter-icons {
+    margin-bottom: 8px;
+  }
```

Comment:
Should we just always use these styles on `.nx-filter-input` rather than having the modifier?

Reply:
> Should we just always use these styles on `.nx-filter-input` rather than having the modifier?

That's a good point: https://github.com/sonatype/sonatype-react-shared-components/pull/166/commits/be56b60a75173f1a2b96d26caf903e832bba40d7